corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -1076,9 +1081,8 @@ def _migrate_case_attachments(couch_case, sql_case):
             name=name or attachment.identifier,
             case=sql_case,
             content_type=attachment.server_mime,
-            content_length=attachment.content_length,
-            blob_id=blob.id,
-            blob_bucket=couch_case._blobdb_bucket(),
```

Comment:
was the `blob_bucket` arg not necessary?

Reply:
No, it's included in `blob.key`